package-build.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kctrl.carvel.dev/v1alpha1
+kind: PackageBuild
+metadata:
+  name: secretgen-controller.carvel.dev
+spec:
+  release:
+  - resource: {}
+  template:
+    spec:
+      app:
+        spec:
+          deploy:
+          - kapp: {}
+          template:
+          - ytt:
+              paths:
+              - config/package-bundle/config
+              - tmp/output-config.yml
+          - kbld:
+              paths:
+              - '-'
```

Comment:
Did we check how to handle generating two images while making a release?

Reply:
Yeah as discussed now I have started passing images.yml in kbld step.